### ISSUE TITLE
Opening Black Isle/Bioware/WotC Logo movies are not replacing Beamdog's

### ISSUE THREAD

**Issue by thejacket** [Sampsca/BGEE-Classic-Movies#10]
Version of game: BGEE
Language: Polish
After installation, old movies are restored in game and I can also choose them in the menu, where I also can find movies in the question.
But is it possible to display them instead or along the new Beamdog's Logo when starting the game?

**Comment by Sampsca** [comment]
Most people want to disable the startup movies for convenience, not add more.  Exactly which logos do you want to add, the same as the original or add some of the oBG2 variants also?

**Comment by thejacket** [comment]
Yes, but enabling/disabling startup movies isn't a part of this mod, since Beamdog and intro cutscene is still present?
I propose the following:
In a WeiDU installer, introduce a separate installation step:
```
Install Component [Replace Beamdog's startup logo with original ones (Black Isle/Bioware/WotC)?]
```

we can maybe instead introduce a choose option installation step:
```
Install Component [Replace Beamdog's startup logo with original ones (Black Isle/Bioware/WotC)?]
1] Replace Beamdog's startup logo with original ones
2] Disable all startup logo movies
3] Disable all startup movies (including intro cutscene)
```

In general, the motivation is that I think that the original logo sequence was super atmospheric, I remember to this day the stormy Black Isle sounds followed by industrial BioWare when I was starting BG in year 2000 and I suspect a lot of people share the same feelings.

**Comment by thejacket** [comment]
I think this is the original intro sequence:
https://www.youtube.com/watch?v=bYoRmGHIxA4

**Comment by Sampsca** [comment]
@thejacket 
The latest commit (1c4caf3323fddad75b6da74a831ae0ece1206320) is a first attempt at implementing this feature request.  I found that in the latest game versions, disabling Beamdog's logo movie also prevented the intro movie from playing.  I have implemented a workaround, but I'm unsure if this behavior has changed since early versions of the game.  I thought I had a BGEE v1.3 laying around for testing purposes, but I was unable to find it.  Try it out and let me know how the new component works.  If all goes well, I'll create a new release once I get translations for the new strings.

**Comment by Sampsca** [comment]
No one has reported this as not working, so it has been added to the latest release.  Closing the issue, but please reopen it if something isn't working as expected.
